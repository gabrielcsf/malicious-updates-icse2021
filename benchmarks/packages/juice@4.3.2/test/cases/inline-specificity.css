.a {
  padding-top: 5px !important;
}
.a {
  padding-bottom: 5px;
}
