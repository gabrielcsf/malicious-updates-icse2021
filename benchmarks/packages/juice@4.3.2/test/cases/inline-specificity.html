<div class="a" style="padding: 10px"></div>
