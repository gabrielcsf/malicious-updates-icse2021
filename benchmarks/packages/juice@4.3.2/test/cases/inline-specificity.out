<div class="a" style="padding-bottom: 5px; padding: 10px; padding-top: 5px;"></div>
